src/index.js

```diff
@@ -1,62 +1,66 @@
-let AWS = require('aws-sdk');
-let fs = require('fs');
-let Group = require('./modules/group');
-let log4js = require('log4js');
-let Membership = require('./modules/membership');
-let MicroCache = require('micro-cache');
-let Search = require('./modules/search');
-let util = require('util');
-
-let FileCertificate = {
+const AWS = require('aws-sdk');
+const fs = require('fs');
+const log4js = require('@log4js-node/log4js-api');
+const util = require('util');
+const _ = require('lodash');
+const MicroCache = require('micro-cache');
+const Group = require('./modules/group');
+const Membership = require('./modules/membership');
+const Search = require('./modules/search');
+
+const FileCertificate = {
   readCertificate: async (opts) => {
-    if (opts.cert === '' || opts.key === '' ||
-      !fs.existsSync(opts.cert) || !fs.existsSync(opts.key)) {
+    if (opts.cert === '' || opts.key === ''
+      // eslint-disable-next-line security/detect-non-literal-fs-filename
```

Comment:
maybe you could eslint disable for the entire file?

Reply:
If I was using more frequently, I might, but It's contained in one block. I prefer to minimize unintended disabling of checks.